src/main/java/org/shanerx/tradeshop/objects/ListManager.java

```diff
@@ -70,14 +72,17 @@ public boolean isDirection(BlockFace face) {
 		return directions.contains(face);
 	}
 
-	public boolean isInventory(Material mat) {
-		return inventories.contains(mat);
+    public boolean isInventory(Block block) {
+        if (!(block instanceof Container))
+            return false;
+
+        return inventories.contains(block.getClass().getSimpleName().replaceAll("[\\[\\]]", "").toLowerCase());
```

Comment:
So that we dont need to have a list we physically have to update with each version (like how the plugin works for 1.14 but not for things like barrels which had been added but we didnt add it to our internal list)

Reply:
Still, we shouldn’t switch to reflection. I don’t think it is actually necessary, it is bothersome to add stuff to the list, but it would still be better

Reply:
Would reflection only on plugin init be better?

Reply:
I suppose it would be better...
It would still be far from ideal though. We could keep it as a temporary solution, but we should change it in the future I think

Reply:
Agreed, will make the change later this week and keep an eye out for a better way to generate the list.

Reply:
This is no longer using reflection